docs/BasicCommunication/OnSystemTimeReady/index.md

```diff
@@ -0,0 +1,37 @@
+## OnSystemTimeReady
+
+Type
+: Notification
+
+Sender
+: HMI
+
+Purpose
+: Inform SDL about readiness to provide system time
+
+!!! must
```

Comment:
Can we replace !!! must with 'The HMI is required to:"

Reply:
@mrapitis, taking into account [smartdevicelink/sdl_markdown_spec](https://github.com/smartdevicelink/sdl_markdown_spec) , I doubt.
@jacobkeeler, please advise.

Reply:
Thanks for the feedback. Agreed after reviewing the spec, please disregard.